docker/Dockerfile

```diff
@@ -307,7 +306,7 @@ RUN chmod +x ./gaia-linux-amd64 \
     && chmod 600 /root/.ssh
 
 # Set homepath as volume
-VOLUME [ "${GAIA_HOMEPATH}" ]
+VOLUME [ "${GAIA_HOME-PATH}" ]
```

Comment:
`-` in environment properties is not a good thing as they usually can't be parsed nicely or cause problems on the command line in ZSH they even have to be explicitly escaped sometimes like `[` or `{`. I would just leave this out. It's fine like that.

Reply:
We need some way to separate longer property options. My first idea was to use `_` but if you pass parameters to the binary it also looks odd: `-home_path`

Reply:
Oh hang on that stuff is auto-read. ARGH. Okay, never mind then.
Also, yeah it should be `--home-path`

Reply:
I found out that `flag` supports that out of the box by replacing `-` with `_` for environment variables. Perfect 😄

Reply:
Oh very awesome! :D :D Perfect. :)